scripts/export.sh

```diff
@@ -35,7 +35,7 @@ echo "Running daemon..."
 lily daemon \
     --repo="${REPO_PATH}" \
     --config=/lily/config.toml \
-    --bootstrap=false &> lily.log &
+    --bootstrap=false > >(tee lily.log) 2&>1 &
```

Comment:
This doesn't work. You need to do `&> >(tee lily.log) &`

Reply:
thanks, I changed it